src/main/java/nl/esciencecenter/xenon/adaptors/schedulers/JobQueueScheduler.java

```diff
@@ -472,9 +458,11 @@ public QueueStatus getQueueStatus(String queueName) throws XenonException {
     }
 
     public void end() {
```

Comment:
Method is only used in `src/test/java/nl/esciencecenter/xenon/adaptors/schedulers/JobQueueSchedulerTest.java`.
The tests should be refactored to use try with resources and then the end method should be removed

Reply:
Will do

Reply:
Done. Tests are running now

Reply:
Nice